src/acl/AllOf.cc

```diff
@@ -56,13 +57,8 @@ Acl::AllOf::parse()
     } else if (oldNode) {
         // this acl saw a single line before; create a new OR inner node
 
-        MemBuf wholeCtx;
-        wholeCtx.init();
-        wholeCtx.appendf("(%s lines)", name);
-        wholeCtx.terminate();
-
         Acl::OrNode *newWhole = new Acl::OrNode;
-        newWhole->context(wholeCtx.content(), oldNode->cfgline);
+        newWhole->context(ToSBuf('(', name, " lines)"), oldNode->cfgline);
```

Comment:
This and similar name-generating lines are a bit difficult to read and write correctly. I am pretty sure we should add a couple of functions or a simple class that encapsulates this logic. However, IIRC, there are related XXXs/bugs that claim that wrong generated names are printed in mgr:config reporting or some other context. I think improving this generation code (and addressing those concerns or ruling them out of scope) deserves a dedicated PR.

For now, we are preserving the current official format for these generated/implicit ACL names and just converting them to SBuf (instead of using MemBuf to assemble them).